lib/order.rb

```diff
@@ -0,0 +1,34 @@
+class Order
+attr_accessor :lines, :total
+
+  def initialize
+    @lines = []
+    @total = 0
+  end
+
+  def add(line)
+    lines.push(line)
+  end
+
+  def totalNoTax
+    lineTotals = self.lines.map {|line| line[1]* line[2]}
```

Comment:
Surrounding space missing for operator '*'.<br>Space between { and | missing.<br>Space missing inside }.<br>Use snake_case for variable names.